### PR TITLE
feat: model management 

### PR DESCRIPTION
**Pull request by Aralhi**
- models shows, create and start, stop

### CODE REVIEW

File: components/model/model-card.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { IModelData } from '@/types/model';
+import { useTranslation } from 'react-i18next';
+import SentimentSatisfiedAltIcon from '@mui/icons-material/SentimentSatisfiedAlt';
+import SentimentVeryDissatisfiedIcon from '@mui/icons-material/SentimentVeryDissatisfied';
+import StopCircleIcon from '@mui/icons-material/StopCircle';
```
Comment (2976151305) [inline components/model/model-card.tsx:6]: mui will be deprecated soon，please choose the icon in `@antd/icons`

File: components/layout/left-side.tsx
Comment (2976151305) [inline components/layout/left-side.tsx:1]: `left-side.tsx` will be removed in the next version

File: components/model/model-card.tsx (hunk 1 above)
Comment (Aralhi) [reply components/model/model-card.tsx:6]: mui icon is more abundant then antd, how about continuing using it. component can change to antd.

File: components/layout/left-side.tsx
Comment (Aralhi) [reply components/layout/left-side.tsx:1]: well. how about merge this branch to main and change it in next version.

**Review by 2976151305 — APPROVED** [review]

**Pull request merged**